src/backends/ilp-quoter/index.js

```diff
@@ -85,6 +75,10 @@ class ILPQuoter {
                 currencyPair[0] + '/' + currencyPair[1] + '/' + amount +
                 '/' + type
     const result = yield request({ uri, json: true })
+    if (result.statusCode >= 400) {
```

Comment:
@lumberj or @gip do we have a good way to unit test this? I think the tests are using the fixer.io backend, but I could be wrong.